src/sql/__init__.py

```diff
@@ -9,16 +9,16 @@
 ssql_secrets.read("../secrets/ssql.cfg")
 config = {
     "ssh": {
-        "host": ssql_secrets.get("ssh", "host"),
-        "port": int(ssql_secrets.get("ssh", "port")),
-        "user": ssql_secrets.get("ssh", "user_name"),
-        "pass": ssql_secrets.get("ssh", "password"),
+        "host": "130.240.200.96",
+        "port": 22,
+        "user": "amuobi-4",
+        "pass": "vBoMgtRDJQpkWeHN",

```

Comment:
This should not contain your secrets, keep them in the config

Reply:
Fixed.